### PR TITLE
Improve Native time on Daily action

### PR DESCRIPTION
**Pull request by Sgitario**
++ Run only http/http-minimum in Windows native only
++ Disable AnnotationScheduledJobsMySqlQuartzIT on Native as it takes around 30 min to finish
++ There were some duplicated test cases in QuarkusCliVersionIT and QuarkusCliCreateJvmApplicationIT

### CODE REVIEW

**Comment by Sgitario** [comment]
These are the scenario that took longer:

```
Time elapsed: 432.972 s - in io.quarkus.ts.sqldb.sqlapp.Postgresql10DatabaseIT
Time elapsed: 826.375 s - in io.quarkus.ts.sqldb.sqlapp.MariaDB102DatabaseIT
Time elapsed: 328.757 s - in io.quarkus.qe.sqldb.panacheflyway.ApplicationResourceIT
Time elapsed: 354.225 s - in io.quarkus.qe.sqldb.panacheflyway.UserResourceIT
Time elapsed: 1,941.858 s - in io.quarkus.qe.scheduling.quartz.AnnotationScheduledJobsMySqlQuartzIT
Time elapsed: 384.336 s - in io.quarkus.ts.quarkus.cli.QuarkusCliCreateApplicationIT
Time elapsed: 1,217.108 s - in io.quarkus.ts.quarkus.cli.QuarkusCliSpecialCharsIT
Time elapsed: 392.366 s - in io.quarkus.ts.quarkus.cli.QuarkusCliVersionIT
```

The CLI scenarios are building the apps always on Native and that's why they took so much. For now, I think we're good to not run these tests on Native.

**Review by rsvoboda — CHANGES_REQUESTED** [review]
++ Not run CLI jobs on Native
add parallel run for CLI tests

++ Disable AnnotationScheduledJobsMySqlQuartzIT on Native as it takes around 30 min to finish
ok for now, but we should think about a way how to run the tests which were disabled just because they took too much time. Please create issue for that

**Comment by Sgitario** [comment]
> ++ Not run CLI jobs on Native
> add parallel run for CLI tests
> 
> ++ Disable AnnotationScheduledJobsMySqlQuartzIT on Native as it takes around 30 min to finish
> ok for now, but we should think about a way how to run the tests which were disabled just because they took too much time. Please create issue for that

> add parallel run for CLI tests

Let's speak about it in a private chat.

> ok for now, but we should think about a way how to run the tests which were disabled just because they took too much time. Please create issue for that

I will tag those tests and create an issue to think about how to run these scenarios.

**Comment by Sgitario** [comment]
> > ++ Not run CLI jobs on Native
> > add parallel run for CLI tests
> > ++ Disable AnnotationScheduledJobsMySqlQuartzIT on Native as it takes around 30 min to finish
> > ok for now, but we should think about a way how to run the tests which were disabled just because they took too much time. Please create issue for that
> 
> > add parallel run for CLI tests
> 
> Let's speak about it in a private chat.
> 
> > ok for now, but we should think about a way how to run the tests which were disabled just because they took too much time. Please create issue for that
> 
> I will create an issue to think about how to run these scenarios: #154

About the CLI tests, the problem is that when building on Native, all the builds were done in Native but this was not meant to behave like this. Instead, all the tests should be run on JVM but a few, to verify build native. 
I've already updated the scenario to cover JVM and Native in the same time.

File: quarkus-cli/pom.xml
Comment (rsvoboda) [inline quarkus-cli/pom.xml:33]: This can be a bit problem for Jenkins jobs when I expect to have JVM and Native variants of the jobs.

This way we would need to specify Mandrel image for JVM job too ?
And there would be no cli tests executed on NATIVE job?

Comment (Sgitario) [reply quarkus-cli/pom.xml:33]: No CLI tests executed on Native job.
But good point about the Mandrel image.. we're losing this coverage

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliVersionIT.java
Comment (Sgitario) [inline quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliVersionIT.java:47]: I've just realised that these scenarios were duplicated. We should keep the ones in QuarkusCliCreateJvmApplicationIT

**Review by rsvoboda — APPROVED** [review]

**Comment by rsvoboda** [comment]
waiting for CI before merging

**Pull request merged**
